integration-tests/src/test/java/tachyon/client/RemoteBlockInStreamIntegrationTest.java

```diff
@@ -78,19 +78,17 @@ public RemoteBlockInStreamIntegrationTest(String[] classes) {
   @After
   public final void after() throws Exception {
     mLocalTachyonCluster.stop();
-    System.clearProperty(Constants.WORKER_DATA_SERVER);
-    System.clearProperty(Constants.USER_REMOTE_BLOCK_READER);
   }
 
   @Before
   public final void before() throws Exception {
     mLocalTachyonCluster = new LocalTachyonCluster(10000, 1000, Constants.GB);
-    System.setProperty(Constants.WORKER_DATA_SERVER, mDataServerClass);
-    System.setProperty(Constants.USER_REMOTE_BLOCK_READER, mRemoteReaderClass);
     mLocalTachyonCluster.start();
-    mLocalTachyonCluster.getWorkerTachyonConf().set(Constants.USER_REMOTE_READ_BUFFER_SIZE_BYTE,
-        "100");
     mMasterTachyonConf = mLocalTachyonCluster.getMasterTachyonConf();
+    TachyonConf workerTachyonConf = mLocalTachyonCluster.getWorkerTachyonConf();
+    workerTachyonConf.set(Constants.USER_REMOTE_READ_BUFFER_SIZE_BYTE, "100");
+    workerTachyonConf.set(Constants.WORKER_DATA_SERVER, mDataServerClass);
```

Comment:
In [`mLocalTachyonCluster.start()`](https://github.com/amplab/tachyon/pull/1352/files#diff-3468a5e5623a435ee4a657759d2b191bL90), the worker is started, if `WORKER_DATA_SERVER` is specified here, how can a worker decide which data server to use?